src/images/icons/hamburger.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#2a2e36"><path d="M0 0h24v24H0z" fill="none"/><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"/></svg>
```

Comment:
can we use the MUI icon for that as we currently have the MUI package installed for only the icons. It is planned to refactor to a separate only icon libary to reduce the bundle size, but this is not the scope of this PR.

Reply:
done